Sources/Rendering/Core/Mapper/api.md

```diff
@@ -359,3 +359,41 @@ Provide read access to the color texture coordinate array
 
 Provide read access to the color texture array
 
+### viewSpecificProperties
+
+Object that contains specific properties to be able to apply
+your own substitutions to the shader creation process.
+You can apply your own string replacements to add features you desire.
+
+PolyDataMapper is the only object that uses it.
```

Comment:
I would add test here to say, go look in the rendering backend of your choice for details on specific properties.  For OpenGL/WebGL see ...